reframe/frontend/cli.py

```diff
@@ -502,10 +502,16 @@ def main():
 
     # Miscellaneous options
     misc_options.add_argument(
-        '-C', '--config-file', action='append', metavar='FILE',
+        '--config-file', action='append', metavar='FILE',
+        dest='config_file_alias',
+        help="Deprecated, alias of `--config-files'",
+        envvar='RFM_CONFIG_FILE :'
+    )
+    misc_options.add_argument(
+        '-C', '--config-files', action='append', metavar='FILE',
```

Comment:
Now I just realised that this option is `append`, so it can't be named `--config-files`. I thought we were passing all the configs with a single argument. So we can keep the `--config-file` name. Sorry for the back and forth! But using `RFM_CONFIG_FILES` instead of `RFM_CONFIG_FILE` is correct.